tensorboard_plugin_wit/wit_plugin.py

```diff
@@ -318,7 +329,8 @@ def _infer(self, request):
             model_signatures[model_num],
             request.args.get('use_predict') == 'true',
             request.args.get('predict_input_tensor'),
-            request.args.get('predict_output_tensor'))
+            request.args.get('predict_output_tensor'),
+            custom_predict_fn=custom_predict_fn)
```

Comment:
if you set up WIT to have two models, then this would use the same predict fn for both models. instead you need a way to set/store one custom predict fn for the primary model, and separately set a second one if comparing two models. this could be just a function that is custom_wit_predict_fn.custom_compare_predict_fn or similarly named.

Reply:
oops, I didn't know that WIT can compare the result of different models concurrently. Did you mean "ANOTHER MODEL FOR COMPARISON"? How about adding another textbox in the "Set up your data and model" page and let user fill the function names?

Reply:
Yes if the user adds another model through that button, the inference address and model name will be set for that second model in this serving_bundle when calling into the second model.

So your example code that uses the serving bundle info to make the pyserve call should actually work correctness for model comparison as written now. 

You can verify that by trying with two separate pyserve'd models with different results and verifying it works in WIT as expected.